src/guide/mailing-lists.md

```diff
@@ -10,6 +10,10 @@ The mailing lists are the key communications mechanism for all OpenJDK work. All
 
 > [`mail.openjdk.org`](https://mail.openjdk.org/mailman/listinfo)
 
+:::{.note}
+Please note that OpenJDK is about **development** of the JDK, not usage. This is not a place to ask support questions. Please direct those to whomever provided your JDK build. If you think you found a bug in the JDK, see [JBS - JDK Bug System].
+:::
+
```

Comment:
To clarify my earlier comment, I was recommending a link to be included to

https://bugreport.java.com/bugreport/

where people who do not have JBS accounts can initiate a bug being filed.

Reply:
The referenced section (JBS - JDK Bug System) has links to both JBS and Bugreport.